src/OrbitBase/TemporaryFile.cpp

```diff
@@ -44,6 +44,9 @@ ErrorMessageOr<void> TemporaryFile::Init() {
         absl::StrFormat("Unable to create a temporary file: %s", SafeStrerror(errnum))};
   }
 
+  // get rid of '\0' at the end
+  file_path = std::filesystem::path(file_path.string().c_str());
```

Comment:
`file_path `is a string already. Perhaps make a copy of the `file_path `above and append the` \0` to that copy.

Reply:
but then we will need to copy it over.. _mktemp_s modifies the string. Fixed by replacing fs::path with std::string() here.